### ISSUE TITLE
Cleanup more trait bounds

### ISSUE THREAD

**Issue by insipx** [paritytech/substrate-archive#363]
We can clean up trait bounds on our structs that we could not before due to https://github.com/Restioson/xtra/pull/42 being merged into `xtra`

**Comment by dvdplm** [comment]
Is this still relevant? What structs do you have in mind here?

**Comment by insipx** [comment]
Yeah! Still relevant. The PR I Linked allows us to remove the `Send + Sync` bounds from our Actor-struct declarations, like here for example:

https://github.com/paritytech/substrate-archive/blob/ff02024da349eb03b238165c501476343bcab9d5/substrate-archive/src/actors/workers/blocks.rs#L42

which will have a ripple effect and hopefully clean up the code elsewhere as well

**Comment by dvdplm** [comment]
I see, ok. But the PR is not released yet, so we can't do this yet right?

**Comment by insipx** [comment]
yeah we're just waiting on a release of xtra. Not really high on priorities since it won't change of effect the functionality of archive in any way other than improve the code quality
